src/Symfony/Component/Process/Process.php

```diff
@@ -303,6 +303,10 @@ public function start(callable $callback = null, array $env = array())
             $ptsWorkaround = fopen(__FILE__, 'r');
         }
 
+        if (!file_exists($this->cwd)) {
+            @trigger_error('The provided cwd does not exist. Command is currently ran against getcwd()', E_USER_DEPRECATED);
```

Comment:
the message should mention that an exception will be thrown in 4.0

Reply:
Done.